### ISSUE TITLE
Cannot require package.json from Wallaby bootstrap

### ISSUE THREAD

**Issue by endragor** [wallabyjs/public#123]
I'd like to avoid Jest config duplication and use fields from the config in package.json. But for some reason `require('./package.json')` in bootstrap ends up with `Cannot find module` error. Am I missing something?
Can be reproduced in https://github.com/endragor/wallaby-jest-sample

**Comment by ArtemGovorov** [comment]
It may be some Jest integration specific issue, I'll have a look. For now as workaround you may require the file by its full path, like
``` JavaScript
const jestConfig = require(require('path').join(wallaby.projectCacheDir, 'package.json')).jest;
```

**Comment by ArtemGovorov** [comment]
Fixed in core 1.0.51.
